opencti-platform/opencti-front/src/private/components/arsenal/vulnerabilities/VulnerabilityEditionOverview.js

```diff
@@ -242,40 +202,38 @@ class VulnerabilityEditionOverviewComponent extends Component {
         commitMutation({
           mutation: vulnerabilityMutationRelationDelete,
           variables: {
-            id: this.props.vulnerability.id,
+            id: vulnerability.id,
             toId: head(removed).value,
             relationship_type: 'object-marking',
           },
         });
       }
     }
-  }
+  };
 
-  render() {
-    const { t, vulnerability, context, enableReferences } = this.props;
-    const createdBy = convertCreatedBy(vulnerability);
-    const objectMarking = convertMarkings(vulnerability);
-    const status = convertStatus(t, vulnerability);
-    const initialValues = pipe(
-      assoc('createdBy', createdBy),
-      assoc('objectMarking', objectMarking),
-      R.assoc('x_opencti_workflow_id', status),
-      pick([
-        'name',
-        'description',
-        'confidence',
-        'createdBy',
-        'killChainPhases',
-        'objectMarking',
-        'x_opencti_workflow_id',
-      ]),
-    )(vulnerability);
-    return (
+  const createdBy = convertCreatedBy(vulnerability);
+  const objectMarking = convertMarkings(vulnerability);
+  const status = convertStatus(t, vulnerability);
+  const initialValues = pipe(
+    assoc('createdBy', createdBy),
+    assoc('objectMarking', objectMarking),
+    R.assoc('x_opencti_workflow_id', status),
```

Comment:
why R.assoc here and assoc the line just above?

Reply:
No reason, fixed